### PR TITLE
Linux switching for CRT

### PR DESCRIPTION
**Pull request by alphanu1**
## Guidelines

1. Rebase before opening a pull request
2. If you are sending several unrelated fixes or features, use a branch and a separate pull request for each
3. If possible try squashing everything in a single commit. This is particularly beneficial in the case of feature merges since it allows easy bisecting when a problem arises

## Description

[Description of the pull request, detail any issues you are fixing or any features you are implementing]

## Related Issues

[Any issues this pull request may be addressing]

## Related Pull Requests

[Any other PRs from related repositories that might be needed for this pull request to work]

## Reviewers

[If possible @mention all the people that should review your pull request]

### CODE REVIEW

File: gfx/display_servers/dispserv_x11.c [hunk 1]
```diff
@@ -37,6 +44,8 @@ static void* x11_display_server_init(void)
 
 static void x11_display_server_destroy(void *data)
 {
+   
+   system("xrandr -s 704x480");
```
Comment (RobLoach) [inline gfx/display_servers/dispserv_x11.c]: This seems unneeded.

**Comment by alphanu1** [comment]
OK here it is. I have spoken to @hunterk and @markwkidd they suggested to create a pull request with what I have. I know system() functions are not wanted. However, I don't have time to recreate xrandr within RA.

@hunterk said that someone can take this over to finish up as I no longer have the time. I am more than happy to help out whoever tries to change the necessary parts.

Resolution and frequency matching is so close to perfect, much better than windows.

@twinaphex

**Comment by twinaphex** [comment]
We can use the system calls for now until we work out a better solution, HOWEVER, we will need you @alphanu1 to test it once we are there, since I think none of us own a similar setup that allows us to test this.

**Comment by hizzlekizzle** [comment]
I should be able to pitch-in more with the linux testing. I have a few CRTs I can hook up to, I just didn't have any appropriate Windows setups.

**Comment by alphanu1** [comment]
@twinaphex Yeah that is fine. I will be working on a RPi version soon as well.

**Comment by markwkidd** [comment]
I'm going to close the github issue because IMO the criteria for collecting the bounty has been met. That way @alphanu1 can begin the bounty process.

**Comment by twinaphex** [comment]
OK, assuming @hizzlekizzle can directly help out with the Linux CRT SwitchRes side when we refactor this, let's merge it then.

**Comment by alphanu1** [comment]
There is a prerequisite to install one modeline on boot. In this case its 704x480@60 as this is the current exit res @hizzlekizzle. It obviously would be better with a res detection on the first res change to set the orig_width and orig_height like windows, so we can restore to that.  This can be done by creating a .sh making it executable and a startup program.

**Comment by alphanu1** [comment]
To be honest this is probably the best res to have for Linux on a CRT anyway.
